templates/frontend/objects/article_details.tpl

```diff
@@ -129,6 +129,22 @@
 				{/if}
 			{/foreach}
 
+			{* Keywords *}
+			{if !empty($keywords[$currentLocale])}
+			<div class="item keywords">
+				<span class="label">
+					{translate key="article.subject"}
```

Comment:
The keywords label should have a `:` after it like the DOI, eg - `Keywords: <list of keywords>`.